flytekit/engines/unit/engine.py

```diff
@@ -188,6 +203,13 @@ def _execute_user_code(self, inputs):
             )
         return results
 
+    @property
+    def has_workflow_node(self):
```

Comment:
don't think we should expose this as a property since it isn't relevant outside the class. also not sure if the indication of this is best represented as an instance variable.

Reply:
removing the property.  Not quite sure what you mean by the second comment.

Reply:
it's kinda weird to set the value on the instance itself since theoretically you might call it multiple times and it might yield different nodes on each run (like if you were unit testing someday). I think you can just put a `TODO: remove when unit testing is implemented for workflow/launch plan nodes`